### DISCUSSION TITLE
pdf中bookmarks与目录中有细微区别。

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#227] [Q&A] [answered]
您好，

比如在模板中，在`chapter1.tex`文件中输入头两行：

> \section{论文主要部分的写法}\label{sec:chap1}
> \section{对应\ref{sec:chap1}{ }\hyperref[sec:chap1]{节}}

生成的pdf文件目录与pdf书签不一样，改怎么修复pdf书签？
![image](https://github.com/whutug/whu-thesis/assets/45897471/28b550e8-3c38-43d9-8620-63b33c5efe03)

谢谢

**Comment by SwitWu** [comment]
`\hyperref` 命令用法不是这样的

**Reply by earthmoon** [reply]
那应该怎么用？

**Comment by SwitWu** [comment, marked as answer]
```
\section{论文主要部分的写法}\label{sec:chap1}
\section{对应 \texorpdfstring{\hyperref[sec:chap1]{\ref*{sec:chap1} 节}}%
  {\ref{sec:chap1} 节}}
```

**Reply by earthmoon** [reply]
好的，谢谢！
